src/Razor/test/Microsoft.AspNetCore.Razor.Test.ComponentShim/Microsoft.AspNetCore.Components.netstandard2.0.cs

```diff
@@ -207,8 +207,11 @@ public static partial class EventCallbackFactoryEventArgsExtensions
     public sealed partial class EventHandlerAttribute : System.Attribute
     {
         public EventHandlerAttribute(string attributeName, System.Type eventArgsType) { }
+        public EventHandlerAttribute(string attributeName, System.Type eventArgsType, bool enablestopPropagation, bool enablePreventDefault) { }
```

Comment:
I'll make this change in the real type once this change is approved.

Reply:
I'm not convinced we need to add the ability to customize `enablestopPropagation` / `enablePreventDefault` . I'm of the opinion that all `EventHandlerAttribute`s should have this ability (they do already don't they?). 

@rynowak @SteveSandersonMS do either of you feel differently?

Reply:
The point of this change was for the runtime to declare it's support for various features, and for the compiler to light up based on that. I think it's preferrable to hardcoding versions.

Reply:
> The point of this change was for the runtime to declare it's support for various features

Arent these two features synonymous with things like `@bind:culture` (something that's baked in)?